### DISCUSSION TITLE
post/kakaopay-techlog/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#38] [Announcements]
# post/kakaopay-techlog/

카카오페이 기술 블로그를 개발하고 유지보수한 경험에 대해 공유합니다.

https://tech.kakaopay.com/post/kakaopay-techlog/

**Comment by pay-dan-dy** [comment]
기술블로그 플랫폼 구축을 고민하시는 분들께 단비와 같은 내용이 될 것 같습니다. 좋은 글 감사합니다!

**Comment by moonkorea00** [comment]
구멘

**Comment by gyunseo** [comment]
저도 astro로 개인 블로그 운영 중인데, 많은 인싸이트 얻고 갑니다~~! 좋은 글 감사합니다!

**Comment by YieumYoon** [comment]
덕분에 새로운 프레임워크를 알게 되어서 깃헙 블로그에 사용해보려고 합니다. 감사합니다!
